css/style.css

```diff
@@ -801,10 +801,20 @@ div#content
 	background-color: #f6f6f6;
 	font-size: 0.875em;
 	line-height: 1.4em;
-	text-align: justify;
 	border: 0.25em solid #ccd5d5;
 }
 
+/*  This is a CSS hack to only target Blink based browsers
+    Targets Chrome 28 and later and Opera 14 and later
+*/
+@media all and (-webkit-min-device-pixel-ratio:0) and (min-resolution: .001dpcm) {
```

Comment:
Ouch at this hack. There has to be a better way, even if it involves JS. Ideally a bit of JS will detect if hyphenation is actually supported by the browser or not, I think hyphenate.js has such a chechk.